code/modules/implant_hacking/_hacking.dm

```diff
@@ -0,0 +1,162 @@
+/obj/item/cyberlink_connector
+	name = "Wireless personal connector"
+	desc = "Allows you to connect to incompatible implants and hack them."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "connector"
+	var/obj/item/organ/cyberimp/cyberlink/parent_cyberlink
+	var/list/datum/hacking_minigame/game_list = list()
+	var/time_left = 0
+	var/obj/item/organ/cyberimp/cybernetic
+	var/mob/living/current_user
+	var/processing = FALSE
```

Comment:
what? why?

Reply:
we need to keep track if the game is already going to preserve the game inbetween tgui openings and closings, it just points to if the game was already started.